src/Driver/DropzoneUploadDriver.php

```diff
@@ -124,14 +125,17 @@ private function saveMonolith(UploadedFile $file, StorageConfig $config, Closure
      */
     private function saveChunk(UploadedFile $file, Request $request, StorageConfig $config, Closure $fileUploaded = null): Response
     {
-        $numberOfChunks = $request->post('dztotalchunkcount');
-
-        $range = new RequestRange(
-            $request->post('dzchunkindex'),
-            $numberOfChunks,
-            $request->post('dzchunksize'),
-            $request->post('dztotalfilesize')
-        );
+        try {
+            $range = new RequestBodyRange(
+                $request,
+                'dzchunkindex',
+                'dztotalchunkcount',
+                'dzchunksize',
+                'dztotalfilesize'
+            );
+        } catch (InvalidArgumentException $e) {
+            throw new BadRequestHttpException($e->getMessage());
```

Comment:
Might be better here to preserve the exception trace history by passing `$e` to the new exception